using_images/other_images/jenkins_slaves.adoc

```diff
@@ -28,7 +28,7 @@ The first is a link:https://github.com/openshift/jenkins/tree/master/slave-base[
 Two more images that extend the base image are also provided:
 
 * link:https://github.com/openshift/jenkins/tree/master/agent-maven-3.5[Maven v3.5 image]
-* link:https://github.com/openshift/jenkins/tree/master/agent-nodejs-8[Node.js v8 image]
+* link:https://github.com/openshift/jenkins/tree/master/agent-nodejs-12[Node.js v12 image]
```

Comment:
Also "https://github.com/openshift/jenkins/tree/master/agent-nodejs-10[Node.js v10 image]" in addition to 12